src/signing_key.rs

```diff
@@ -42,6 +42,17 @@ use pkcs8::der::pem::PemLabel;
 
 use crate::{VerificationKey, VerificationKeyBytes};
 
+/// The length of a ed25519 `SecretKey`, in bytes.
+pub const SECRET_KEY_LENGTH: usize = 32;
+
+/// ed25519 secret key as defined in [RFC8032 § 5.1.5]:
+///
+/// > The private key is 32 octets (256 bits, corresponding to b) of
+/// > cryptographically secure random data.
+///
+/// [RFC8032 § 5.1.5]: https://www.rfc-editor.org/rfc/rfc8032#section-5.1.5
+pub type SecretKey = [u8; SECRET_KEY_LENGTH];
```

Comment:
These new type aliases and constants are unfortunately useless at the moment. The reason is that all modules here are private and only specific items are re-exported. All new items should be added to `lib.rs` for all this to become usable.